default.nix

```diff
@@ -111,7 +111,7 @@ stdenv.mkDerivation rec {
 
   enableParallelBuilding = true;
 
-  buildFlags = [ "world" ];
+  buildFlags = [ "world" ] ++ optional buildIde "coqide";
```

Comment:
@coq/nix-maintainers should coqide be optional in the dune install call at line 123? (regardless of this PR)

Reply:
I'm not using that Nix config myself but I'd say yes.

Reply:
Indeed, it should. You can do so like this (IINM):

```nix
  installPhase = ''
    runHook preInstall
    dune install --prefix $out --libdir $OCAMLFIND_DESTDIR coq-core coq-stdlib coq coqide-server ${optional buildIde "coqide"}
    runHook postInstall
  '';
```